components/NavigationDropdown.vue

```diff
@@ -34,9 +34,10 @@ export default {
 .navigation-dropdown {
   box-shadow: 0 24px 32px 0 rgba(0, 0, 0, 0.03),
     0 8px 32px 0 rgba(0, 0, 0, 0.06);
-  color: $grey-50;
-  font-size: 0.875rem;
-  font-weight: $font-normal;
+  color: $grey-60;
+  font-size: 14px;
```

Comment:
```suggestion
  font-size: 0.875rem;
```
**Nitpick :** Garder les tailles de font en `rem`